docs/source/deployment/index.rst

```diff
@@ -0,0 +1,43 @@
+===============
+Deploy AiiDAlab
+===============
+
+There are several ways one can deploy AiiDAlab: locally for oneself, on a server for multiple users, or on a Kubernetes cluster for many users and high availability and scalability.
+All these approaches rely on the fact that AiiDAlab is served as a Docker image.
+
+
+***********************
+Deploy AiiDAlab locally
+***********************
+
+This is by far the easiest way of getting started with AiiDAlab.
+First, you need to make sure that Docker is installed on your machine, otherwise, go to `Get Started with Docker <https://www.docker.com/get-started>`__ page and follow the instructions for your operating system.
+Then, start AiiDAlab by running:
+
+   .. code-block:: console
+
+       $ ./run.sh 8888 ~/aiidalab
+
+Feel free to choose a different port and directory.
+
+You should wait untill the starup procedure is completed.
+Once it is done, you can open the link that has been prompted for you at the end of the execution in your browser.
+
+
+**********************
+Deploy AiiDAlab server
+**********************
+
+Deploying AiiDAlab server requires installing several packages such as `JupyterHub <https://jupyter.org/hub>`__, `DockerSpawner <https://github.com/jupyterhub/dockerspawner>`__, `Apache HTTP Server <https://www.apache.org/>`__, and `Docker <http://www.docker.com>`__.
+Further, one needs to configure the interactions between those services.
+The final step is to download the `aiidalab-docker-stack <https://hub.docker.com/repository/docker/aiidalab/aiidalab-docker-stack>`__ Docker image and tag it as `aiidalab-docker-stack:latest`.
```

Comment:
Why does it have to be latest?

Reply:
That is the image name JupyterHub would look for to start a new container.

Reply:
Got it.